packages/base-styles/_variables.scss

```diff
@@ -45,6 +45,7 @@ $grid-unit-80: 8 * $grid-unit;		// 64px
  */
 
 $icon-size: 24px;
+$icon-size-next-default-32px: 32px; // transitionary variable for next small button size
```

Comment:
It looks like `icon-size` is used in a lot of places, and often for icons — I'm not sure that we should change this variable.

What about the `$button-size-small` variable a few lines below? Should we not be using that variable instead, also in the `Button` component?

Reply:
You're absolutely right, my bad 🙈 Thanks for noticing!